mycore-solr/src/main/java/org/mycore/solr/schema/MCRSolrConfigReloader.java

```diff
@@ -89,19 +90,64 @@
     }
 
     /**
-     * This method modified the SOLR config definition based on all solr/{coreType}/solr-config.json 
+     * Clean SOLR configuration for defined items. This cleaning works over all in the property
+     * MCR.Solr.ObserverConfigTypes defined SOLR configuration parts for all in the
+     * MCR.Solr.ObserverConfigTypes.{observedType}.toClean property defined entries. For each entry the 
+     * method will process a SOLR delete command via API.
+     *
+     * @param configType the name of the configuration directory containing the SOLR core configuration
+     * @param coreID the ID of the core, which the configuration should be applied to
+     */
+    public static void cleanConfig(String configType, String coreID) {
+        LOGGER.info("Clean config definitions for core " + coreID + " using configuration " + configType);
+        try (CloseableHttpClient httpClient = HttpClients.createDefault()) {
+            String coreURL = MCRSolrClientFactory.get(coreID)
+                .orElseThrow(() -> MCRSolrUtils.getCoreConfigMissingException(coreID)).getV1CoreURL();
+            List<String> observedTypes = getObserverConfigTypes();
+            JsonObject currentSolrConfig = retrieveCurrentSolrConfig(coreURL);
+            JsonObject configPart = currentSolrConfig.getAsJsonObject("config");
+
+            for (String observedType : observedTypes) {
+                String ignoreListConfiguration = MCRConfiguration2
+                    .getString("MCR.Solr.ObserverConfigTypes." + observedType + ".toClean").orElse("");
+                List<String> cleanList = Arrays.asList(ignoreListConfiguration.split("\\s*,\\s*"));
```

Comment:
I would like to see this as a utitlity method in MCRConfiguration2. getStringList(String propName).
Even if streams are elegant and this code is part of the Javadoc, 
it would simplify the code in modules / application where a list of strings is needed.